src/secrets_manager/secrets_manager.py

```diff
@@ -24,7 +24,7 @@
 def _check_for_kebab_case(secret_name: str) -> None:
     if "-" in secret_name:
         warnings.warn(
-            f"The secret '{secret_name}' should not be kebab case. Use {str(secret_name).replace('-', '_')} instead",
+            f"The secret '{secret_name}' should not be kebab case as environmental variables can only be snake case. Use {str(secret_name).replace('-', '_')} instead",
```

Comment:
technically "snake_case" is lowercase, and CONSTANT_CASE is uppercase---both are acceptable (it's the underscore vs hyphen that is the big issue).  not a high priority to correct though